app/services/container_deployment_service.rb

```diff
@@ -54,6 +54,6 @@ def optional_vms(vms)
   end
 
   def optional_deployment_types
-    %w(openshiftOrigin, openshiftEnterprise)
+    %w(openshiftOrigin openshiftEnterprise)
```

Comment:
@dkorn it seems weird to see this different casing here.

Also, this list should probably constructed in a different way (rather than hard-coded) but I wouldn't go into it right now.
For example: I assume that you have this list somewhere else when you handle the yaml generation (check with @alongoldboim )... so I'd start to consolidate that in a proper place.